demo/App/App.jsx

```diff
@@ -0,0 +1,54 @@
+import React, {Component} from 'react';
+import {HashRouter as Router, Route, Link} from 'react-router-dom';
+
+import {Layout, Menu} from 'antd';
+
+import 'antd/lib/layout/style';
+import 'antd/lib/icon/style';
+import 'antd/lib/menu/style';
+
+const {Header, Sider, Content} = Layout;
+
+import DemoPage from './DemoPage';
+
+const examples = schemas.map(item => require(`./../schemas/${item}`));
+
+export default class App extends Component {
+  render() {
+    return (
+      <Router>
+        <Layout>
```

Comment:
Please, set the height of the `Layout` component to 100% (`<Layout style={{height: '100%'}}>`).